MainApp/public/app/controllers/gamePageCtrl.js

```diff
@@ -1,74 +1,204 @@
-angular.module('gamePageControllers', ['timer'])
-    .controller('gamePageCtrl', function ($scope, $http, $routeParams, $timeout) {
-		let app = this;
-     	app.username = $routeParams.username;
-     	
-     	$scope.counter = "10:00";
-     	let seconds = 600;
-
-        let roomSelector = "div[class='panel-body fixed-panel center']";
-     	let div1 = document.getElementById("R1");
-    	let div2 = document.getElementById("R2");
-    	let div3 = document.getElementById("R3");
-    	let div4 = document.getElementById("R4");
-    	let div5 = document.getElementById("R5");
-    	let div6 = document.getElementById("R6");
-
-    	let map = new Map();
-    	let patientMap = new Map();
-
-    	map.set("div1", "green") 
-    	map.set("div2", "green")
-    	map.set("div3", "green")
-    	map.set("div4", "green")
-    	map.set("div5", "green")
-    	map.set("div6", "green")
-
-    	patientMap.set("div1", null) 
-    	patientMap.set("div2", null)
-    	patientMap.set("div3", null)
-    	patientMap.set("div4", null)
-    	patientMap.set("div5", null)
-    	patientMap.set("div6", null)
-
-    	let patients = 0;
-    	let doctors = 0;
-    	let surgeons = 0;
-    	let patientSelected;
-
-
-  		$("#patients").click(function () {
-  			$('#patientsgroup').show();
-  			$('#patients').hide();
-  			$('#resourcesGroup').hide();
-  			$('#resources').show();
-
-  		});
-
-  		$('#resources').click(function () {
-  			$('#resources').hide();
-  			$('#resourcesGroup').show();
-  			$('#patients').show();
-  			$('#patientsgroup').hide();
-
-  		})
+angular.module('gamePageControllers', ['roomServices'])
+    .controller('gamePageCtrl', function ($scope, $http, $routeParams, $timeout, PatientService, Room, Agent) {
+        let app = this;
+        let patientService = PatientService;
+
+       
+        let statsObject = {};
+        statsObject.finalScore = 10;
+        statsObject.username = "Syed";
```

Comment:
No. It's just hard coded. We are in the process of writing a gameState function wherein you get the username along with score and resource count that writes in the userStatistics collection. Just for an end to end flow I hard coded. :)